roles/wildfly_systemd/tasks/main.yml

```diff
@@ -203,6 +203,9 @@
         - daemon_reload is defined or daemon_reload_env is defined
         - daemon_reload.changed or daemon_reload_env.changed
 
+    - name: "Ensure service is started"
```

Comment:
@RanabirChakraborty you copy/paste the next tasks and forget to change the content of the name.